src/daml/Daml/Finance/Setup/Scenario/BondIssuance.daml

```diff
@@ -0,0 +1,119 @@
+
+  -- Create clock and events
+  let clock = DateClock with u = Unit today, id = Id "Clock", provider = issuer, observers = fromList pub
+  submitMulti [issuer] [] do createCmd clock
+  submitMulti [issuer] [] do createCmd DateClockUpdateEvent with clock; id = Id $ "Update to " <> show today
+
+  -- Create accounts
+  accountFactoryCid <- toInterfaceContractId <$> createFactory operator pub
+  holdingFactoryCid <- toInterfaceContractId <$> submit operator do createCmd Fungible.Factory with provider = operator; observers = fromList pub
+  let createAcc = createAccount [public] accountFactoryCid holdingFactoryCid
+  issuerCashAccount      <- createAcc [] centralBank issuer    "Issuer@CentralBank"
+  custodianCashAccount   <- createAcc [] centralBank custodian "Custodian@CentralBank"
+  aliceCashAccount       <- createAcc [] custodian   alice     "Alice@Custodian/Cash"
+  bobCashAccount         <- createAcc [] custodian   bob       "Bob@Custodian/Cash"
+  charlieCashAccount     <- createAcc [] custodian   charlie   "Charlie@Custodian/Cash"
+  issuerSecAccount       <- createAcc [] registry    issuer    "Issuer@Registry"
+  custodianSecAccount    <- createAcc [] registry    custodian "Custodian@Registry"
+  aliceSecAccount        <- createAcc [] custodian   alice     "Alice@Custodian/Securities"
+  bobSecAccount          <- createAcc [] custodian   bob       "Bob@Custodian/Securities"
+  charlieSecAccount      <- createAcc [] custodian   charlie   "Charlie@Custodian/Securities"
+
+  -- Instruments
+  usd   <- originate         centralBank centralBank pub todayTime "USD"
+  bond1 <- originateZeroCouponBond      registry issuer "BOND1" "ZERO-3Y"       pub todayTime (date 2022 Sep todayDay) (date 2025 Sep todayDay) usd
+  bond2 <- originateFixedRateBond       registry issuer "BOND2" "FIXED-3Y-1.1%" pub todayTime (date 2021 Sep todayDay) ["FED"] registry (date 2022 Sep todayDay) (date 2025 Sep todayDay) Act365Fixed Following 0.011 M 12 usd
+  bond3 <- originateFloatingRateBond    registry issuer "BOND3" "FIXED-3Y-0.8%" pub todayTime (date 2021 Sep todayDay) ["FED"] registry (date 2022 Sep todayDay) (date 2025 Sep todayDay) Act365Fixed Following 0.005 M 12 usd "USD-LIBOR-12M"
+  bond4 <- originateInflationLinkedBond registry issuer "BOND4" "INFL-3Y-0.75%" pub todayTime (date 2021 Sep todayDay) ["FED"] registry (date 2022 Sep todayDay) (date 2025 Sep todayDay) Act365Fixed Following 0.005 M 12 usd "CPI" 100.0
+
+  -- issue operator registry issuer public "ISSUE1" (Instrument.qty 100_000_000.0 bnd1) issuerSecAccount
+  -- issue operator registry issuer public "ISSUE2" (Instrument.qty 100_000_000.0 bond2) issuerSecAccount
+  -- issue operator registry issuer public "ISSUE3" (Instrument.qty 100_000_000.0 bond3) issuerSecAccount
+  -- issue operator registry issuer public "ISSUE4" (Instrument.qty 100_000_000.0 bond4) issuerSecAccount
+
+  -- deposit operator registry custodian public (Instrument.qty 100_000_000.0 bond1) custodianSecAccount
```

Comment:
If we don't need these lines, better to remove them (doesn't need to be done as part of this PR)